otx/algorithms/common/adapters/mmcv/nncf/hooks.py

```diff
@@ -24,6 +27,17 @@ def before_run(self, runner):
         if runner.rank == 0:
             runner.logger.info(self.compression_ctrl.statistics().to_str())
 
+    @master_only
+    def after_run(self, runner):
+        compression_state = self.compression_ctrl.get_compression_state()
+        for algo_state in compression_state.get("ctrl_state", {}).values():
+            if not algo_state.get("scheduler_state"):
+                algo_state["scheduler_state"] = {"current_step": 0, "current_epoch": 0}
```

Comment:
Needed but sometimes missing?

Reply:
Not sure this is sometimes missing I just copied it from an original code block.